styles/main.css

```diff
@@ -687,3 +705,19 @@ form .cta-button:hover {
     line-height: 20px;
   }
 
+  .contact {
+    background: #0f27ff url("../images/bg-contact-desktop.png") no-repeat;
+    background-size: 100%;
+    border-radius: 100px 1px 1px 1px;
+    padding: 2% 25%;
+    justify-content: center;
+  }
```

Comment:
- [x] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. The background color and background images of your design are appearing differently from the template. Please fix the background color and images and make sure it matches the design.

Your design

![Screenshot (121)](https://user-images.githubusercontent.com/83803414/154155862-11493862-856c-4078-b825-eb8c5bef6c5f.png)


Figma design

![Screenshot (122)](https://user-images.githubusercontent.com/83803414/154155844-773cbfdb-93dc-406a-9cb6-54fbabde267c.png)